ui/components/src/BarChart/BarChart.tsx

```diff
@@ -104,6 +105,10 @@ export function BarChart(props: BarChartProps) {
         formatter: (params: { name: string; data: number[] }) =>
           params.data[1] && `<b>${params.name}</b> &emsp; ${formatValue(params.data[1], unit)}`,
       },
+      grid: {
```

Comment:
can you add a comment here why this is needed?

Reply:
Sure! [caae104](https://github.com/perses/perses/pull/1325/commits/caae1049f423d9b6bde6d35ac829883109239b7d)